AutomaticInterface/AutomaticInterface/Builder.cs

```diff
@@ -17,11 +17,14 @@ private static string InheritDoc(ISymbol source) =>
             genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
             memberOptions: SymbolDisplayMemberOptions.IncludeParameters,
             parameterOptions: SymbolDisplayParameterOptions.IncludeType
-                | SymbolDisplayParameterOptions.IncludeParamsRefOut,
+                | SymbolDisplayParameterOptions.IncludeParamsRefOut
+                | SymbolDisplayParameterOptions.IncludeDefaultValue
+                | SymbolDisplayParameterOptions.IncludeName,
```

Comment:
I've expanded the parameter formatting rules so we can use this format for parameter generation.